### ISSUE TITLE
Output to text files

### ISSUE THREAD

**Issue by ch40s** [Dracovian/Discord-Scraper#28]
It would be nice to have the option to store the messages in text files under each server, channel, YYYY-MM-DD directory instead of just the database.

**Comment by Dracovian** [comment]
I did make an attempt at storing the data in a text document and it ended up being a mess to maintain.

I suppose I could make it so that it just writes the raw JSON data to the hard drive since that's what the Discord site returns.

**Comment by ch40s** [comment]
Yes, I think it would be nice to have it as an option in config.json.

**Comment by Dracovian** [comment]
I won't ping you twice on the same issue, just want to make sure you give the experimental branch a test to see if it is the solution you were looking for.

Otherwise feel free to open this issue again (same title is fine) but with the Experimental tag assigned to it so that I can be assured that the issue is for the experimental branch code and not the master branch code.

**Comment by ch40s** [comment]
@Dracovian thanks! I can't assign labels/tabs though, it might be a permissions issue.

**Comment by Dracovian** [comment]
@ch40s I suppose it looks like Github only allows for moderators and owners of repos to set issue labels... I imagine that was a feature by now but apparently not.

Well now I know, and I'm going to continue working on with the experimental script in the meantime.

**Comment by ch40s** [comment]
@Dracovian the script is not creating any files in the same directory while running in loop.

**Comment by Dracovian** [comment]
@ch40s Is there a Text directory along with a Scrapes directory?

**Comment by ch40s** [comment]
@Dracovian no, they are not being created by the script.

**Comment by Dracovian** [comment]
@ch40s I did find that I shouldn't have added the *"&has=image&has=videos"* to the query because it skips over the text data. I did that to improve the original script's performance before I added the text grabbing feature.

It's difficult to figure a way to separate queries to improve runtime performance without having to send an excess of requests to the server (which the script already does from poor design).

Hopefully these changes should bring about more results from the text scraping function.

**Comment by ch40s** [comment]
@Dracovian the directories and files are now being created but they are empty, they don't include any messages.
